applications/dashboard/models/class.usermodel.php

```diff
@@ -2932,18 +2932,24 @@ public function saveIP($userID, $IP, $dateUpdated = false) {
 
     /**
      * Updates visit level information such as date last active and the user's ip address.
-     *
-     * @param int $UserID
-     * @param string|int|float $ClientHour
+     * @param $UserID
+     * @param int|float $ClientHour
+     * @throws Exception
```

Comment:
The parameters are self-explanatory without needing a description, but I'd like to see one for the exception.  Maybe something like:

```
* @throws Exception if the user ID is not valid.
```